chain/bitcoind_conn.go

```diff
@@ -167,54 +184,92 @@ out:
 		// Poll an event from the ZMQ socket. It's possible that the
 		// connection to the socket continuously times out, so we'll
 		// prevent logging this error to prevent spamming the logs.
-		msgBytes, err := zmqConn.Receive()
+		msgBytes, err := conn.Receive()
 		if err != nil {
 			err, ok := err.(net.Error)
 			if !ok || (ok && !err.Timeout()) {
 				log.Error(err)
 			}
 
-			continue out
+			continue
 		}
 
-		// We have an event! We'll now determine which kind of event was
-		// read, deserialize it, and report it to the different rescan
+		// We have an event! We'll now ensure it is a block event,
+		// deserialize it, and report it to the different rescan
 		// clients.
-		switch string(msgBytes[0]) {
-		case "rawtx":
-			tx := &wire.MsgTx{}
+		if string(msgBytes[0]) == "rawblock" {
```

Comment:
Should have an else case for logging purposes.

Reply:
Fixed.